src/component-base.js

```diff
@@ -60,7 +61,8 @@ export default class ComponentBase extends LitElement {
       relative:         { type: Boolean },
       secondary:        { type: Boolean },
       ondark:           { type: Boolean },
-      type:             { type: String }
+      type:             { type: String },
+      referrerpolicy:   { type: Boolean }
```

Comment:
Looks like API.md wasn't generated with this new property.

Reply:
https://github.com/AlaskaAirlines/auro-hyperlink/blob/9dbca0843024ad240a9944dc8854fbd7c2dc649d/package.json#L121-L127

I want to let this ride. From the code above it is supposed to run in the remote and then commit back to the repo.